src/benchmarklib/tpch/tpch_benchmark_item_runner.cpp

```diff
@@ -70,27 +71,36 @@ bool TPCHBenchmarkItemRunner::_on_execute_item(const BenchmarkItemID item_id, Be
 }
 
 void TPCHBenchmarkItemRunner::on_tables_loaded() {
-  if (!_use_prepared_statements) return;
+  // Make sure that sort order, indexes, and constraints have made it all the way up to here
+  const auto orders_table = Hyrise::get().storage_manager.get_table("orders");
+  const auto first_chunk = orders_table->get_chunk(ChunkID{0});
+  Assert(first_chunk->ordered_by(), "Ordering information was lost");
+  if (_config->indexes) {
+    Assert(!first_chunk->get_indexes({ColumnID{0}}).empty(), "Index was lost");
+  }
+  Assert(!orders_table->get_soft_unique_constraints().empty(), "Constraints were lost");
 
-  std::cout << " - Preparing queries" << std::endl;
+  if (_use_prepared_statements) {
```

Comment:
From here until the end, nothing changed. I converted the guard clause into a regular if statement as it is not really a guard clause in the strictest sense.